src/stactools/ecmwf_forecast/kerchunk_helper_functions.py

```diff
@@ -0,0 +1,60 @@
+from kerchunk.combine import MultiZarrToZarr
+from kerchunk.grib2 import scan_grib
+from numcodecs.registry import register_codec
+from stactools.ecmwf_forecast.range_codec import Range
+
+import base64
+import fsspec
+
+register_codec(Range)
+
+def get_kerchunk_indices(part):
+
+    #clear instance cache, prevents memory leak
+    fs = fsspec.filesystem('')
+    fs.clear_instance_cache()
+
+    out = scan_grib(part.filename)
+
+    if (((part.stream=='scda')|(part.stream=='oper')) & (part.type=='fc')):
```

Comment:
nitpick: use `or` and `and` instead of `|` and `&`.

Reply:
Done.